lib/actions.js

```diff
@@ -86,23 +86,38 @@ async function main(projectName, options) {
     ).projectName;
   }
 
-  projectName = projectName.toLowerCase().replace(/ |_/g, '-');
+  setProjectName =
+    projectName === '.'
+      ? path.resolve(process.cwd()).toLowerCase().replace(/ |_/g, '-')
```

Comment:
Here we want the name of the current directory right? 

I think this will do the work.
```suggestion
      ? path.basename(process.cwd())
```

Reply:
Hey, I just checked this, this is creating a new project with "projectName" in the "projectName" folder
![Screenshot (289)](https://user-images.githubusercontent.com/43727167/103135403-20afac80-46de-11eb-9823-bbe9a681ff20.png)

Reply:
oh ok let it be then I will check later